inst/qml/doeAnalysis.qml

```diff
@@ -0,0 +1,164 @@
+import QtQuick 2.0
+
+import QtQuick 									2.8
+import QtQuick.Layouts 							1.3
+import JASP.Controls 							1.0
+import JASP.Widgets 							1.0
+
+Form
+{
+    columns:                                    1
```

Comment:
tabs, not spaces

Reply:
I can't find out how to use Qt to align the qml files with tabs instead of spaces on Mac... this is also relevant for doeFactorial